executor/src/witgen/machines/block_machine.rs

```diff
@@ -687,17 +460,17 @@ struct DefaultSequenceIterator {
     current_round_count: usize,
 }
 
-const MAX_ROUNDS_PER_ROW_DELTA: usize = 20;
+const MAX_ROUNDS_PER_ROW_DELTA: usize = 100;
 
 impl DefaultSequenceIterator {
     fn new(block_size: usize, identities_count: usize) -> Self {
-        let block_size_i64 = block_size as i64;
+        let max_row = block_size as i64 - 1;
```

Comment:
As far as I can see, we used to process even the first row of the next block. I don't think this is necessary, and none of the tests fail.